QUEST_LV_0100.tsv

```diff
@@ -5179,20 +5179,20 @@ QUEST_LV_0100_20150317_005178	The scripture and was only able to inflict a wound
 QUEST_LV_0100_20150317_005179	{memo X}
 QUEST_LV_0100_20150317_005180	The Paladin Master asked you to clear the area while he focuses on the crystals. Defeat the Throneweaver coming to attack you. 
 QUEST_LV_0100_20150317_005181	Defeat the Throne Weaver
-QUEST_LV_0100_20150317_005182	Defeat the contaminated mausoleum guardians
-QUEST_LV_0100_20150317_005183	Guardian statue says you can't change back the contaminated guardians. Defeat it instead. 
-QUEST_LV_0100_20150317_005184	Defeat the mausoleum guardians 
-QUEST_LV_0100_20150317_005185	Read the mausoleum gravestone
-QUEST_LV_0100_20150317_005186	There is a guardian statue emitting strange forces. Read the statue.
-QUEST_LV_0100_20150317_005187	Defeat the contaminated masoleum guardians
-QUEST_LV_0100_20150317_005188	The statue contaminated with evil force is summoning guardians. Defeat the contaminated guardians.
+QUEST_LV_0100_20150317_005182	$Defeat the corrupted Mausoleum protectors
+QUEST_LV_0100_20150317_005183	$The Protector Statue said you can't change back corrupted protectors. Defeat them instead. 
+QUEST_LV_0100_20150317_005184	$Defeat the Mausoleum protectors 
+QUEST_LV_0100_20150317_005185	$Read the Mausoleum gravestone
+QUEST_LV_0100_20150317_005186	$There is a Protector Statue emitting strange forces. Read the statue.
+QUEST_LV_0100_20150317_005187	$Defeat the corrupted Mausoleum protectors
```

Comment:
Defeat the corrupted Mausoleum protectors -> Defeat the corrupted Royal Mausoleum guardians (or Guardians?)